testsuite/helloworld/tests/sandbox_c_format_open/format_open.c

```diff
@@ -0,0 +1,17 @@
+#include <errno.h>
+#include <fcntl.h>
+#include <stdio.h>
+#include <string.h>
+#include <unistd.h>
+
+int main() {
+    int fd;
+    if ((fd = open("%s", O_RDONLY)) >= 0) {
+        close(fd);
```

Comment:
We're about to shut down anyway, no need to close.

Reply:
Copied from `nametoolong.c`, but fair, will change.